pkgs/tools/misc/kdecoration-viewer/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, cmake, kde5, qtquickcontrols }:
```

Comment:
Instead of importing `kde5`, you can import `ecm`, `qtquickcontrols`, etc. directly. Then you won't need the `with kde5;` after `buildInputs`. Also, `kde5.callPackage` will ensure that all your dependencies are compatible. Your package doesn't have many dependencies, so it's probably not a problem, but it could prevent problems in the future.